integration/src/main/java/scripts/testscripts/RemoveUser.java

```diff
@@ -25,24 +26,23 @@
 import java.util.List;
 import java.util.UUID;
 import org.apache.commons.lang3.RandomStringUtils;
-import scripts.utils.BqDataTableUtils;
-import scripts.utils.BqDatasetUtils;
-import scripts.utils.ClientTestUtils;
-import scripts.utils.CloudContextMaker;
-import scripts.utils.GcsBucketObjectUtils;
-import scripts.utils.GcsBucketUtils;
-import scripts.utils.NotebookUtils;
-import scripts.utils.WorkspaceAllocateTestScriptBase;
+import org.broadinstitute.dsde.workbench.client.sam.api.GroupApi;
+import scripts.utils.*;
```

Comment:
nit as a convention, we don't have imports with `*`. In the Terra onboarding guide, search for "java imports". That tells you the Intellij setting to change, to avoid `*`.